tools/topology/topology2/cavs-mixin-mixout-hda.conf

```diff
@@ -54,8 +55,9 @@ Object.Pipeline {
 			}
 
 			Object.Widget.gain.1 {
+				use_topology_name "true"
 				Object.Control.mixer.1 {
-					name '1 2nd Playback Volume'
+					name 'Pre Mixer $ANALOG_PLAYBACK_PCM Volume'
```

Comment:
pre- and post mix is fine, but I don't get what ANALOG means here?

Reply:
That refers to the PCM this pipeline is going to.